### PR TITLE
Switch to @shopify/babel-preset and update @babel/core

### PR DESCRIPTION
**Pull request by atesgoral**
## Description

- Update to the latest presets to take advantage of modern features
- Use optional chaining in appropriate places, because now we can

### CODE REVIEW

File: packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export function GraphQLUniversalProvider<
     const requestIdLink = requestID
       ? createRequestIdLink(requestID)
       : undefined;
-    const finalLink = clientOptions.link ? clientOptions.link : undefined;
+    const finalLink = clientOptions.link || undefined;
```
Comment (atesgoral) [inline packages/react-graphql-universal-provider/src/GraphQLUniversalProvider.tsx:46]: Slipped this in along the way

File: package.json [hunk 2]
```diff
@@ -34,11 +34,15 @@
...
     "packages/*"
   ],
+  "babel": {
+    "presets": ["@shopify/babel-preset/web"]
+  },
   "devDependencies": {
     "@apollo/react-common": "^3.1.3",
     "@apollo/react-hooks": "^3.1.3",
-    "@babel/core": "^7.5.5",
+    "@babel/core": "7.11.6",
```
Comment (gmcgibbon) [inline package.json:43]:
```suggestion
    "@babel/core": "^7.11.6",
```

File: package.json [hunk 3]
```diff
@@ -34,11 +34,15 @@
...
     "@shopify/app-bridge": "^0.7.3",
+    "@shopify/babel-preset": "23.1.1",
```
Comment (gmcgibbon) [inline package.json:45]:
```suggestion
    "@shopify/babel-preset": "^23.1.1",
```

**Review by gmcgibbon — APPROVED** [review]
I think the lockfile is out of sync. LGTM once CI is happy 👍

**Comment by atesgoral** [comment]
@gmcgibbon Updated lockfile

**Pull request merged**